test/scripts/test_ping.py

```diff
@@ -1,13 +1,24 @@
 from topology_helper import Topology
 from serial_helper import SerialHelper
-from util import format_node_id_to_hex_str
+from util import format_node_id_to_hex_str, retry_test
 
 
 class TestPing:
     """Ping test class"""
 
     TEST_STRING = "test_string"
 
+    def __compare_data(self, ser: SerialHelper):
```

Comment:
I'm curious why this function has a leading double underscore. Dunderscore causes some specific class-private (not in subclasses) name-mangling behavior.

If there's not some more specific reason than signaling that this is private, use a single underscore, or even none, since there wouldn't inherently be a problem with calling this method externally.

Reply:
This was just to indicate that this is private. I will update.